dataframes/03-from-pandas-to-dask.ipynb

```diff
@@ -0,0 +1,891 @@
+       "      <td>Norbert</td>\n",
+       "      <td>0.343440</td>\n",
+       "      <td>-0.787839</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                       id     name         x         y\n",
+       "timestamp                                             \n",
+       "2000-01-01 00:00:00  1053    Edith -0.082485  0.126788\n",
+       "2000-01-01 00:00:01   992    Zelda -0.292002  0.814259\n",
+       "2000-01-01 00:00:02   991   Yvonne -0.435019 -0.923308\n",
+       "2000-01-01 00:00:03   998   Xavier -0.164077  0.195232\n",
+       "2000-01-01 00:00:04   989  Norbert  0.343440 -0.787839"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pdf = ddf.compute()  # create a pandas dataframe\n",
+    "print(type(pdf))\n",
+    "pdf.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have both `dataframes` we can start to compair the interactions with them"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Conceptual shift - from Update to Insert/Delete\n",
+    "Dask does not update - thus no arguments such as \"inplace= True\" which exist in Pandas.  \n",
+    "For more detials see [issue#653 on github](https://github.com/dask/dask/issues/653)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.1 Rename"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Pandas\n",
+    "print(pdf.columns)\n",
+    "pdf.rename(columns={'id':'ID','name':'Name','x':'coor_x', 'y':'coor_y'},inplace=True)\n",
+    "pdf.columns"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Dask\n",
+    "# Must update using the correct sequence of the columns\n",
+    "print(ddf.columns)\n",
+    "ddf.columns = ['ID','Name','coor_x','coor_y']\n",
+    "ddf.columns"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1.2 Column munipilations  \n",
+    "There are several diffrences when manimuplating data. mose of which require to ran a couple of lines (instead of one-liners)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Convert index into Time column"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Pandas\n",
+    "pdf = pdf.assign(Time=pd.to_datetime(pdf.index).time)\n",
+    "pdf.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Dask\n",
+    "ddf = ddf.assign(Time=ddf.index)\n",
+    "ddf['Time'] = ddf['Time'].dt.time\n",
```

Comment:
Why split this across multiple lines? Does the pandas version not work?

Reply:
This doesn't work on a `dask` dataframe